src/domains/messages-app/07_function_pushnotif.tf

```diff
@@ -290,7 +290,7 @@ resource "azurerm_monitor_autoscale_setting" "push_notif_function" {
         direction = "Decrease"
         type      = "ChangeCount"
         value     = "1"
-        cooldown  = "PT20M"
+        cooldown  = "PT10M"
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏It took me a bit to get how to read those rules, as I've been far from these configurations for some time. Also, Azure docs are a bit misaligned with terraform API: https://learn.microsoft.com/en-us/azure/azure-monitor/autoscale/autoscale-understanding-settings.

As far as I get it, it reads as _"scale up or down based on the average number of requests in the last 15 minutes; add or remove an instance every 10 minutes"_. Am I right?

I advocate adding a comment above each rule, to explain it in plain English. So that readers can grasp the intended behavior.